hacking/build_library/build_ansible/command_plugins/release_announcement.py

```diff
@@ -294,12 +57,21 @@ def init_parser(cls, add_parser):
         parser.add_argument("--twitter-out", type=str, default="-",
                             help="Filename to place the twitter announcement into")
 
-    @staticmethod
-    def main(args):
+    @classmethod
+    def main(cls, args):
+        if sys.version_info < (3, 6):
+            raise errors.DependencyError('The {0} subcommand needs Python-3.6+'
+                                         ' to run'.format(cls.name))
+
+        # Import here because these functions are invalid on Python-3.5 and the command plugins and
+        # init_parser() method need to be compatible with Python-3.4+ for now.
+        # Pylint doesn't understand Python3 namespace modules.
+        from .. announce import create_short_message, create_long_message  # pylint: disable=relative-beyond-top-level
```

Comment:
Syntax error?
```suggestion
        from ..announce import create_short_message, create_long_message  # pylint: disable=relative-beyond-top-level
```

Reply:
I don't believe so.  It should work with any number of spaces in there.